### ISSUE TITLE
Wizard for adding a diagram is not accessible

### ISSUE THREAD

**Issue by seiferma** [Cooperate-Project/CooperateModelingEnvironment#196]
The wizard for adding a diagram is not completely accessible: The contents of the field for entering the model path are not read by the screenreader NVDA. Try to reproduce the issue and look for other places where this issue might exist.

In general, it might be good to redesign the dialog because it is not easy to understand. One idea could be splitting the selection of project and model into two separate controls.

**Comment by johannary** [comment]
The same problem exists in the wizard for creating a project. Everything is read at first, but if trying to tab through a second time the descriptor is read as "Eingabefeld schreibgeschützt leer".  This creates a problem in the CDO connection Tab as the Focus is automatically on the Finish button and the Descriptor is never read.
